lib/pbio/platform/debug/pbdrvconfig.h

```diff
@@ -48,11 +48,18 @@
 #define PBDRV_CONFIG_UART_STM32F4_LL_IRQ_NUM_UART   (1)
 
 #define PBDRV_CONFIG_HAS_PORT_1 (1)
+#define PBDRV_CONFIG_HAS_PORT_A (1)
```

Comment:
Is port A replacing port 1? If so, we should remove all of the port 1 stuff.

Reply:
I'm not actually sure wat the debug port should or shouldn't be like, or whether we still use it. This was the minimal change to get the CI to pass it.

Reply:
This was set up to use a UART on the dev board so that we could actually connect sensors to the dev board and debug them.

Reply:
I revised it to not touch debug/tests at all, so this is resolved.